src/Sulu/Bundle/AdminBundle/Resources/public/js/aura_extensions/sulu-content.js

```diff
@@ -155,37 +163,45 @@ define(function() {
          * Handles the content part of the layout object.
          *
          * @param {Object} content The content config object.
+         * @param {Boolean} defaults Iff true default actions will be executed
          */
-        handleLayoutContent = function(content) {
+        handleLayoutContent = function(content, defaults) {
             var width = content.width,
-                leftSpace = !!content.leftSpace,
-                rightSpace = !!content.rightSpace,
-                topSpace = !!content.topSpace;
-            this.sandbox.emit('sulu.app.change-width', width);
+                leftSpace = (defaults) ? !!content.leftSpace : content.leftSpace,
+                rightSpace = (defaults) ? !!content.rightSpace : content.rightSpace,
+                topSpace = (defaults) ? !!content.topSpace : content.topSpace;
+            if (defaults === true || !!width) {
+                this.sandbox.emit('sulu.app.change-width', width, defaults);
+            }
             this.sandbox.emit('sulu.app.change-spacing', leftSpace, rightSpace, topSpace);
-            this.sandbox.emit('sulu.app.toggle-shrinker', false);
+            if (defaults === true || typeof content.shrinkable !== 'undefined') {
+                this.sandbox.emit('sulu.app.toggle-shrinker', !!content.shrinkable);
+            }
         },
 
         /**
          * Handles the sidebar part of the layout object.
          *
          * @param {Object} sidebar The sidebar config object. If false sidebar gets hidden.
+         * @param {Boolean} defaults Iff true default actions will be executed
          */
-        handleLayoutSidebar = function(sidebar) {
+        handleLayoutSidebar = function(sidebar, defaults) {
```

Comment:
i think "useDefaukts"/"applyDefaults" would be a better name for the second parameter